examples/discovery-search-app/cypress/integration/multi_select_facets/spec.ts

```diff
@@ -0,0 +1,190 @@
+describe('Multi-Select Facets', () => {
+  beforeEach(() => {
+    //TODO: we'll need to make sure we can pass in an override for the query, as we want specific facets to come back
+    // Sets up and handles the collections, component settings, and initial query requests that run on page-load
+    cy.server();
+    cy.fixture('collections/collections.json').as('collectionsJSON');
+    cy.route('GET', '**/collections?version=2019-01-01', '@collectionsJSON').as('getCollections');
+
+    //override the standard query with one that has more facets
+    cy.fixture('query/facetsQuery.json').as('facetsQueryJSON');
+    cy.route('POST', '**/query?version=2019-01-01', '@facetsQueryJSON').as('postQueryFacets');
+
+    //override the component settings to make facets single-select
```

Comment:
Do you mean `multi-select` in this comment too? (btw, I find the comments helpful)

Reply:
yeah -- forgot to change that 😄